.ocamlformat

```diff
@@ -1,8 +1,7 @@
-version=0.13.0
+version=0.14.0
 break-separators=before
 dock-collection-brackets=false
 break-sequences=true
-doc-comments=before
```

Comment:
Since `doc-comments` is set to `before` by default from 0.14.0, we can remove it.
https://discuss.ocaml.org/t/ann-ocamlformat-0-14-0/5435